Source/ASDisplayNode.mm

```diff
@@ -574,10 +570,8 @@ - (UIView *)_locked_viewToLoad
     _viewBlock = nil;
     _viewClass = [view class];
   } else {
-    if (!_viewClass) {
-      _viewClass = [self.class viewClass];
-    }
-    view = [[_viewClass alloc] init];
+    Class c = _viewClass ?: [_ASDisplayView class];
+    view = [[c alloc] init];
```

Comment:
Previously `_viewClass` would be `nil` until the view loaded, then it would go to `_ASDisplayView`. Now `_viewClass` stays nil unless they used a custom view class.